package/skeleton-init-finit/skeleton-init-finit.mk

```diff
@@ -82,15 +82,7 @@ endef
 SKELETON_INIT_FINIT_POST_INSTALL_TARGET_HOOKS += SKELETON_INIT_FINIT_SET_DROPBEAR
 endif
 
-ifeq ($(BR2_PACKAGE_FRR),y)
-define SKELETON_INIT_FINIT_SET_FRR
-	for svc in babeld bfdd bgpd eigrpd isisd ldpd ospfd ospf6d pathd ripd ripng staticd vrrpd zebra; do	\
-		cp $(SKELETON_INIT_FINIT_AVAILABLE)/frr/$$svc.conf $(FINIT_D)/available/$$svc.conf;		\
-	done
-	ln -sf ../available/zebra.conf $(FINIT_D)/enabled/zebra.conf
-endef
 SKELETON_INIT_FINIT_POST_INSTALL_TARGET_HOOKS += SKELETON_INIT_FINIT_SET_FRR
```

Comment:
Shouldn't this line be removed too?

Reply:
That part has been reverted now.
Now process monitor is handled by finit instead of watchfrr.